CONTRIBUTING.md

```diff
@@ -33,109 +41,204 @@ For an introduction to the basics of reST syntax, the source format in which Spy
 
-* The ``master`` branch contains the in-development documentation for Spyder 5; changes specific to that version should be made against this branch.
-* The ``4.x`` branch contains the docs for Spyder 4 and is still maintained, which all PRs not specifically for Spyder 5 should be based against.
-* The ``3.x`` branch is frozen, containing the docs for the legacy Spyder 3 version; no further PRs will be accepted
+For non-trivial changes, its easy to work with Spyder-Docs locally by following a few simple steps.
 
+**Note**: You may need to substitute ``python3`` for ``python`` in the commands below on some Linux distros where ``python`` isn't mapped to ``python3`` (yet).
 
 
-## Submitting Pull Requests
+### Fork and clone the repo
 
-We welcome contributions from the community, and will do our best to review all of them in a timely fashion.
-To do so, please submit a pull request (PR) to this repo against the appropriate branch with your changes, and create a corresponding issue as well if your change is substantive, so that we can keep track of everything and give you credit for closing it.
+First, navigate to the [project repository](https://github.com/spyder-ide/spyder-docs) in your web browser and press the ``Fork`` button to make a personal copy of the repository on your own Github account.
+Then, click the ``Clone or Download`` button on your repository, copy the link and run the following on the command line to clone the repo:
 
-Please make sure your PR titles are brief but descriptive, and include ``PR:`` as a prefix; most importantly, make sure you follow and fill out the template provided, which should guide you through the process and make sure everything runs smoothly.
+```shell
+git clone <LINK-TO-YOUR-REPO>
+```
 
+Finally, set the upstream remote to the official Spyder-Docs repo with:
 
+```shell
+git remote add upstream https://github.com/spyder-ide/spyder-docs.git
+```
 
-## Building and Testing Locally
 
-For non-trivial changes, its easy to work with Spyder-Docs locally by following a few simple steps.
+### Create and activate a fresh environment
 
+We highly recommend you create and activate a virtual environment to avoid any conflicts with other packages on your system or causing any other issues.
+Of course, you're free to use any environment management tool of your choice (conda, virtualenvwrapper, pyenv, etc).
 
-### Install dependencies
+To do so with Conda (recommended), simply execute the following:
 
-Make sure you have the appropriate dependencies installed in your active Python environment for the script to work.
-You can install them into your current conda environment with:
+```shell
+conda create -c conda-forge -n spyder-docs-env python=3.9
+```
+
+And activate it with
+
+```shell
+conda activate spyder-docs-env
+```
+
+With pip/venv, you can create a virtual environment with
+
+```shell
+python -m venv spyder-docs-env
+```
+
+And activate it with the following on Linux and macOS,
```

Comment:
maybe it has sense to have creation + activation per environment management (?)

Reply:
I'm a little unsure of what you're trying to say here, but I *think* you might be saying we should have two separate sections, one per environment management tool? I implemented that, and I agree its a big improvement, thanks!

Reply:
Looks like this was resolved, as discussed above.